.travis.yml

```diff
@@ -1,70 +1,57 @@
 # Travis can building for Linux and macOS
-matrix:
+language: python
+addons:
+  apt:
+    update: true
+    packages: libxkbcommon-x11-0  # Linux package needed for Qt to work
+  homebrew:
+    update: true
+    packages: freetype
+
+services: xvfb
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+jobs:
+  allow_failures:
+    - python: 3.8
+    - os: osx
   include:
-    - os: linux
-      dist: xenial 
-      sudo: required
-      language: python
-      python: 3.5
-    - os: linux
-      dist: xenial 
-      sudo: required
-      language: python
-      python: 3.6
-    - os: linux
-      dist: xenial
-      sudo: required
-      language: python
-      python: 3.7
-    # To maximise compatibility pick earliest image, OS X 10.12
+    - python: 3.8
+      before_install: pip install pygame==2.0.0.dev6
```

Comment:
Wondering if we could declare this in `setup.py` instead, where all dependencies are set. Otherwise, we risk going back to scattered around `pip installs` for dependencies which, in the long run, make life more difficult. ;)

Reply:
Python 3.8 is purely experimental here (jobs: allow_failures: - python: 3.8) and I felt is right to keep all the experimentation in one place.  We do not support Py38 in setup.py's trove classifiers and putting a ___dev___ version of pygame into setup.py did not seem right.

Reply:
Understood. I would then add a VERY VISIBLE comment saying something like that and advising moving such dependencies into `setup.py` when the time comes. :)